spawn_sdk/spawn_sdk/lib/system/spawn_system.ex

```diff
@@ -291,21 +292,6 @@ defmodule SpawnSdk.System.SpawnSystem do
 
   defp call_instance(instance, command, value, context) do
     instance.handle_command({parse_command(command), unpack_unknown(value)}, context)
-  rescue
```

Comment:
this is capturing wrong implementation errors on actors and making it VERY confuse to debug